plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -2000,5 +2001,41 @@ var _ = Describe("validator", func() {
 				Expect(err).To(HaveOccurred())
 			})
 		})
+
+		Context("shoot deletion", func() {
+			var (
+				shootStore             cache.Store
+				intCoreInformerFactory coreinformers.SharedInformerFactory
```

Comment:
There is already such var in L49 (`coreInformerFactory coreinformers.SharedInformerFactory`). Is it possible to reuse it?